blog/zh/blog/2023/10/09/wecity-uses-apisix.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "荷兰智慧城市革命：WeCity 与 APISIX 的合作之旅"
+authors:
+  - name: Arjen Hof
+    title: Author
+    url:
+    image_url:
+  - name: Tim van Densen
+    title: Author
+    url:
+    image_url:
+  - name: Jing Yan
+    title: Technical Writer
+    url:
+    image_url: https://avatars.githubusercontent.com/u/114121331?v=4
+keywords:
```

Comment:
Update keyword to align with the en-us post. 

https://github.com/apache/apisix-website/pull/1691/files#diff-7848529cce3ac2c4956e64335588284851bb6b0c2979540a04ecabc4b18195b3L12-L16